roles/network_plugin/kube-ovn/templates/cni-kube-ovn.yml.j2

```diff
@@ -608,3 +608,19 @@ spec:
   ports:
     - port: 10665
       name: metrics
+{% if kube_ovn_ic_enable %}
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovn-ic-config
```

Comment:
This pull request adds the configmap `ovn-ic-config`, but I cannot find any places where use the configmap.
Could you explain how to use the configmap for supporting OVN interconnect?

Reply:
The configmap is used by kube-ovn in https://github.com/kubeovn/kube-ovn/blob/master/pkg/controller/ovn-ic.go. 
It's documented in https://kubeovn.github.io/docs/v1.11.x/en/advance/with-ovn-ic/

Reply:
Thanks for your reply.
https://kubeovn.github.io/docs/v1.11.x/en/advance/with-ovn-ic/ explains it well.

/lgtm